vcr/ambassador.go

```diff
@@ -90,7 +90,7 @@ func (n ambassador) Start() error {
 		network.MaxReprocessBufferSize)
 	conn, _, err := n.eventManager.Pool().Acquire(context.Background())
 	if err != nil {
-		return fmt.Errorf("failed to subscribe to REPROCESS event stream: %v", err)
+		return fmt.Errorf("time-out acquiring NATS connection from pool: %w", err)
```

Comment:
This is weird:

- is a timeout the only error you can get?
- why a timeout if the given context is Background?

Reply:
Time-out was the error returned during test, but with this "REPROCESS" error log. But looking at the code, I can't really find out how it is triggered, due to the flow of `Acquire`. I think it's more complex than it should be, will make an issue for refactoring it. I'll revert these error messages for now.